### PR TITLE
Revert "refactor!: rename Fusion to Hilla"

### PR DESCRIPTION
**Pull request by platosha**
Reverts vaadin/flow#12672

The rename turned out to cause too much hassle. After the revert, the plan is to introduce the endpoint artifact in the separate Hilla repo and deprecate this one.

### CODE REVIEW

**Review by haijian-vaadin — APPROVED** [review]

File: fusion-endpoint/src/main/java/com/vaadin/fusion/EndpointNameChecker.java [hunk 1]
```diff
@@ -51,7 +51,8 @@
                     "true", "try", "typeof", "var", "void", "volatile", "while",
                     "with", "yield")));
 
-    private static final Pattern WHITESPACE_PATTERN = Pattern.compile("\\s");
+    private static final Pattern CONTAINS_WHITESPACE_PATTERN = Pattern
+            .compile(".*[\\s+].*");
```
Comment (platosha) [inline fusion-endpoint/src/main/java/com/vaadin/fusion/EndpointNameChecker.java:55]:
```suggestion
    private static final Pattern WHITESPACE_PATTERN = Pattern.compile("\\s");
```

File: fusion-endpoint/src/main/java/com/vaadin/fusion/EndpointNameChecker.java [hunk 2]
```diff
@@ -68,7 +69,7 @@ public String check(String endpointName) {
         if (ECMA_SCRIPT_RESERVED_WORDS.contains(endpointName)) {
             return "Endpoint name cannot be equal to JavaScript reserved words";
         }
-        if (WHITESPACE_PATTERN.matcher(endpointName).find()) {
+        if (CONTAINS_WHITESPACE_PATTERN.matcher(endpointName).matches()) {
```
Comment (platosha) [inline fusion-endpoint/src/main/java/com/vaadin/fusion/EndpointNameChecker.java:72]:
```suggestion
        if (WHITESPACE_PATTERN.matcher(endpointName).find()) {
```

File: fusion-endpoint/src/main/java/com/vaadin/fusion/generator/SchemaGenerator.java [hunk 3]
```diff
@@ -265,7 +265,6 @@ private void addFieldAnnotationsToSchema(FieldDeclaration field,
 
     private boolean isAnnotationIndicatingRequired(String str) {
         return str.matches("(NonNull|NotNull|NotEmpty|NotBlank)\\(.+")
-                || (str.matches("Size\\(.+")
-                        && Pattern.compile("\\bmin:[^0]").matcher(str).find());
+                || str.matches("Size\\(\\{.*min:[^0].+");
```
Comment (platosha) [inline fusion-endpoint/src/main/java/com/vaadin/fusion/generator/SchemaGenerator.java:268]:
```suggestion
                || (str.matches("Size\\(.+")
                        && Pattern.compile("\\bmin:[^0]").matcher(str).find());
```

**Review by platosha — COMMENTED** [review]
Let’s keep regex improvements though.

File: fusion-endpoint/src/main/java/com/vaadin/fusion/generator/SchemaGenerator.java [hunk 4]
```diff
@@ -26,7 +26,6 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
```
Comment (platosha) [inline fusion-endpoint/src/main/java/com/vaadin/fusion/generator/SchemaGenerator.java:28]:
```suggestion
import java.util.Set;
import java.util.regex.Pattern;
```

**Pull request merged**